### PR TITLE
Replace assume_role() with aws_assume_role_lib()

### PR DESCRIPTION
**Pull request by kbalk**
Replaces custom boilerplate code for assuming a role with a library found on github.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -4,7 +4,7 @@ terraform {
 }
 
 locals {
-  name = "new_account_iam_role_${random_string.id.result}"
+  name = "new-account-iam-role-${random_string.id.result}"
```
Comment (lorengordon) [inline main.tf:7]: This is a breaking change and will cause resource cycles. We missed the window to align the separator across these projects. It's not a terribly big deal if it's different from one project to the next, thankfully.

File: save/new_account_iam_role.py [hunk 2]
```diff
@@ -0,0 +1,326 @@
+#!/usr/bin/env python3
```
Comment (lorengordon) [inline save/new_account_iam_role.py:1]: This `save/` directory seems like it doesn't belong. Accidentally included in a commit?

**Review by lorengordon — COMMENTED** [review]
Oh it's much easier to see the changes in this PR. Pretty much good to go, just a couple seemingly unrelated items snuck in that we don't need to do.

File: main.tf (hunk 1 above)
Comment (kbalk) [reply main.tf]: ok

File: save/new_account_iam_role.py (hunk 2 above)
Comment (kbalk) [reply save/new_account_iam_role.py]: ok

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+### 0.3.0
+
+**Commit Delta**: [Change from 0.2.0 release](https://github.com/plus3it/terraform-aws-org-new-account-iam-role/compare/0.2.0...0.3.0)
+
+**Released**: 2022.04.22
```
Comment (lorengordon) [inline CHANGELOG.md:11]:
```suggestion
**Released**: 2021.04.22
```

**Review by lorengordon — APPROVED** [review]
rock and roll :shipit:

File: CHANGELOG.md (hunk 3 above)
Comment (kbalk) [reply CHANGELOG.md]: ok

**Review by lorengordon — APPROVED** [review]

**Pull request merged**